src/py/flwr/server/strategy/fast_and_slow.py

```diff
@@ -320,6 +320,7 @@ def aggregate_fit(
             # Track contributions to the global model
             for client, fit_res in results:
                 cid = client.cid
+                assert fit_res.num_examples_ceil is not None
```

Comment:
Intentional to keep the asserts here?

Reply:
Yes - this is not ideal, but part of the migration away from the deprecated values inside FitRes and EvaluateRes. The next step (i.e., one of the upcoming releases) will remove these properties and migrate to custom key/value pairs in the `dict` returned by `fit`/`evaluate`, which means that the strategies need to be (a) refactored to pull the necessary values from the `dict`, and (b) communicate what kind of key/value pairs each strategy expected to do their job.